REV20231005F/BOM_RevF.csv

```diff
@@ -0,0 +1,73 @@
+Quantity,Supplier Part Number 1,Designator,Comment,Value,Footprint
+
+"48","507-0ZCM0004FF2GCT-ND","RT1A, RT1B, RT1C, RT1D, RT2A, RT2B, RT2C, RT2D, RT3A, RT3B, RT3C, RT3D, RT4A, RT4B, RT4C, RT4D, RT5A, RT5B, RT5C, RT5D, RT6A, RT6B, RT6C, RT6D, RT7A, RT7B, RT7C, RT7D, RT8A, RT8B, RT8C, RT8D, RT9A, RT9B, RT9C, RT9D, RT10A, RT10B, RT10C, RT10D, RT11A, RT11B, RT11C, RT11D, RT12A, RT12B, RT12C, RT12D","0ZCM0004FF2G","","PTCC1607X06N"
+"1","68AH8094","PICOZED1","7Z030-PicoZed","",""
```

Comment:
It looks like we don't have the Picozed's part number recorded correctly. Where is `68AH8094` coming from? I don't get anything if I search Octopart for it.

The [AMDC Docs](https://docs.amdc.dev/hardware/subsystems/picozed.html) provides a link to the [`7Z030 version of the Picozed`](https://www.avnet.com/shop/us/products/avnet-engineering-services/aes-z7pz-7z030-som-i-g-rev-e-3074457345635303359), which indicates a part number of `AES-Z7PZ-7Z030-SOM-I-G/REV-E`. Seems like that is what we should be using?

Reply:
Hello @elsevers, I updated Picozed part number as per your suggestion. Please review and give your feedback